src/components/SearchForm/SearchForm.js

```diff
@@ -11,11 +19,12 @@ function Search(props) {
           type="text"
```

Comment:
* You need to wrap any inputs in tag `form` to make it possible to submit and handle inputs correctly